int64 rightWheel
int64 leftWheel
int64 servo1
int64 servo2
int64 servo3